build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -x
+set -e 
+cd harmonize \
+    && sbt assembly \
```

Comment:
![image](https://user-images.githubusercontent.com/965232/54451197-abe9c900-4728-11e9-8579-fb52d97be5d1.png)

Reply:
Missing Confluent resolver in build.sbt. Should be fine now.